website/lang/spanish.js

```diff
@@ -28,6 +28,26 @@ export default {
       anyDistance: "Cualquier distancia",
       xDistance: "{distance} millas",
     },
+    filter: {
+      appointmentType: {
+        title: "Tipo de cita",
+        options: ["Todas las dosis", "Segunda dosis solamente"],
+      },
+      vaccineType: {
+        title: "Tipo de vacuna",
+        options: [
+          "Todos",
+          "Johnson and Johnson",
+          "Moderna",
+          "Pfizer",
+          "Unknown",
```

Comment:
Should "Unknown" be something different in spanish?

Reply:
Yes,  that escaped my attention. Just fixed it.